testdata/project-v3-with-deploy-image/config/samples/example.com_v1alpha1_memcached.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: example.com.testproject.org/v1alpha1
+kind: Memcached
+metadata:
+  name: memcached-sample
+spec:
+  # TODO(user): edit the following value to ensure the number 
+  # of Pods/Instances your Operand must have on cluster        
+  size: 1
+
+  # TODO(user): edit the following value to ensure the the container has the right command
+  # and parameters to be initialized 
+  containerCommand: 'memcached -m=64 modern -v'
```

Comment:
We need to fix this one in the sample testdata and e2e tests to be  'memcached -m=64 -o modern -v'